src/state/opamUpdate.ml

```diff
@@ -67,7 +67,7 @@ let repository rt repo =
     in
     OpamProcess.Job.with_text text @@
     OpamRepository.update r repo_root @@+ fun has_changes ->
-    let has_changes = if redirect then `Changes else has_changes in
+    let has_changes = if redirect then `Changes [] else has_changes in
```

Comment:
That seems unnecessary. @rjbou do you know why this line was added in https://github.com/ocaml/opam/pull/5146 ?